bower.json

```diff
@@ -7,17 +7,36 @@
   "license": "http://polymer.github.io/LICENSE.txt",
   "private": true,
   "dependencies": {
-    "platinum-sw": "polymerelements/platinum-sw#^1.3.0",
-    "paper-styles": "polymerelements/paper-styles#^1.0.0",
-    "iron-icons": "polymerelements/iron-icons#^1.0.0",
-    "iron-icon": "polymerelements/iron-icon#^1.0.0",
+    "platinum-sw": "polymerelements/platinum-sw#2.0-preview",
+    "app-storage": "polymerelements/app-storage#2.0-preview",
+    "paper-styles": "polymerelements/paper-styles#2.0-preview",
+    "iron-icons": "polymerelements/iron-icons#2.0-preview",
+    "iron-icon": "polymerelements/iron-icon#2.0-preview",
     "font-roboto": "polymerelements/font-roboto#^1.0.0",
-    "iron-flex-layout": "polymerelements/iron-flex-layout#^1.0.0",
-    "paper-button": "polymerelements/paper-button#^1.0.0",
-    "paper-fab": "polymerelements/paper-fab#^1.0.0",
-    "paper-icon-button": "polymerelements/paper-icon-button#^1.0.0",
+    "iron-flex-layout": "polymerelements/iron-flex-layout#2.0-preview",
+    "paper-button": "polymerelements/paper-button#2.0-preview",
+    "paper-fab": "polymerelements/paper-fab#2.0-preview",
+    "paper-icon-button": "polymerelements/paper-icon-button#2.0-preview",
     "web-animations-js": "web-animations/web-animations-js#^2.2.1",
-    "iron-input": "polymerelements/iron-input#^1.0.0",
-    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#^1.0.0"
+    "iron-input": "polymerelements/iron-input#2.0-preview",
+    "iron-autogrow-textarea": "polymerelements/iron-autogrow-textarea#2.0-preview"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
```

Comment:
Can you add `"app-storage": "polymerelements/app-storage#^1.0.0",` to this variant? `polymer install --variants` fails without it since it tries to install `2.0-preview` along with all these other dependencies.